<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoukStatusView</class>
 <widget class="QWidget" name="DoukStatusView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DoukMemEd: Status View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lblPlayerPosComplex">
     <property name="text">
      <string>Player Position (Complex) goes here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnReadMem">
     <property name="text">
      <string>Read State...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vHP">
     <item>
      <layout class="QHBoxLayout" name="hMaxHP">
       <item>
        <widget class="QLabel" name="lMaxHP">
         <property name="text">
          <string>Maximum HP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbMaxHP">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="displayIntegerBase">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hCurHP">
       <item>
        <widget class="QLabel" name="lCurHP">
         <property name="text">
          <string>Current HP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbCurHP">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="displayIntegerBase">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLocks">
     <item>
      <widget class="QCheckBox" name="cbLockHP">
       <property name="text">
        <string>Lock HP (uses Current HP value)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbInfBoost">
       <property name="text">
        <string>Infinite Booster Fuel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbInfAmmo">
       <property name="text">
        <string>Infinite Ammo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hEquips">
     <item>
      <layout class="QVBoxLayout" name="vEquips1">
       <item>
        <widget class="QCheckBox" name="cbEquip01">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip02">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip03">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip04">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vEquips2">
       <item>
        <widget class="QCheckBox" name="cbEquip05">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip06">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip07">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip08">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vEquips3">
       <item>
        <widget class="QCheckBox" name="cbEquip09">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip10">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vEquips4">
       <item>
        <widget class="QCheckBox" name="cbEquip13">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip14">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip15">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEquip16">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
